src/app/manager-dashboard/manager-dashboard.component.html

```diff
@@ -35,9 +35,10 @@ <h3 i18n *ngIf="showParentList">{{ planetType === 'community' ? 'Nation' : 'Cent
     <p><b i18n>Local Version:</b> {{versionLocal}}</p>
     <ng-container [ngSwitch]="requestStatus">
       <ng-container *ngSwitchCase="'accepted'">
+        <p><b i18n> List all from the nation</b></p>
```

Comment:
it can even be center so needs to be dynamic. Also we can change words to sound better